subprojects/enterprise-operations/src/main/java/org/gradle/api/internal/tasks/execution/ResolveTaskMutationsBuildOperationType.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.tasks.execution;
+
+import org.gradle.internal.operations.BuildOperationType;
+
+/**
+ * Represents resolving the mutations of a task.
+ * <p>
+ * Before task execution ({@link org.gradle.api.internal.tasks.execution.ExecuteTaskBuildOperationType}),
+ * the mutation of the tasks need to be resolved. The mutations of the task are the locations it may modify,
```

Comment:
Typo, should be `the mutationS of the tasks` in the first sentence.
I also think it should be `The mutations of A task are` in the second sentence.

Reply:
Changed.